src/ObjectUtils/PdbFileTest.h

```diff
@@ -50,27 +51,133 @@ TYPED_TEST_P(PdbFileTest, LoadDebugSymbols) {
     symbol_infos_by_address.emplace(symbol_info.address(), &symbol_info);
   }
 
-  EXPECT_EQ(symbol_infos_by_address.size(), 5469);
+  /*for (auto[address, symbol] : symbol_infos_by_address) {
+      ORBIT_LOG("0x%x: %s", address, symbol->name());
+  }*/
+
+  ASSERT_EQ(symbol_infos_by_address.size(), 5459);
 
   {
-    const SymbolInfo& symbol = *symbol_infos_by_address[0x18000ef90];
+    const SymbolInfo& symbol = *symbol_infos_by_address[0x18000eea0];
     EXPECT_EQ(symbol.name(), "PrintHelloWorldInternal");
-    // TODO(b/219413222): We actually also expect the parameter list (empty in this case), but the
-    //  DIA SDK implementation does not support this yet.
-    EXPECT_TRUE(absl::StrContains(symbol.demangled_name(), "PrintHelloWorldInternal"));
-    EXPECT_EQ(symbol.address(), 0x18000ef90);
+    EXPECT_EQ(symbol.demangled_name(), "PrintHelloWorldInternal()");
+    EXPECT_EQ(symbol.address(), 0x18000eea0);
     EXPECT_EQ(symbol.size(), 0x2b);
   }
 
   {
-    const SymbolInfo& symbol = *symbol_infos_by_address[0x18000efd0];
+    const SymbolInfo& symbol = *symbol_infos_by_address[0x18000eee0];
     EXPECT_EQ(symbol.name(), "PrintHelloWorld");
-    // TODO(b/219413222): We actually also expect the parameter list (empty in this case), but the
-    //  DIA SDK implementation does not support this yet.
-    EXPECT_TRUE(absl::StrContains(symbol.demangled_name(), "PrintHelloWorld"));
-    EXPECT_EQ(symbol.address(), 0x18000efd0);
+    EXPECT_EQ(symbol.demangled_name(), "PrintHelloWorld()");
+    EXPECT_EQ(symbol.address(), 0x18000eee0);
     EXPECT_EQ(symbol.size(), 0xe);
   }
+
+  {
+    const SymbolInfo& symbol = *symbol_infos_by_address[0x18000ef00];
```

Comment:
We're potentially dereferencing a nullptr here, I'd prefer asserting that the map contains the address we're looking for so that it's clearer if the test fails. Something like:

```
const SymbolInfo* symbol = symbol_infos_by_address[0x18000ef00];
ASSERT_NE(symbol, nullptr);
EXPECT_EQ(symbol->name(), "PrintString");
```

Same for other occurrences of that pattern in the file.